user_tasks/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 from django.dispatch import Signal
 
-__version__ = '1.1.0'
+__version__ = '1.2.0'
```

Comment:
Please update the changelog also (it looks like that was forgotten for the last release too).

Reply:
Sure.